### ISSUE TITLE
Scarabs missing from fragments tab

### ISSUE THREAD

**Issue by Stickymaddness** [Stickymaddness/Procurement#918]
I've just tested the changes from #910, and can not see _any_ of the scarabs I have. I can still see other fragments such as sacrifice fragments, divine vessel etc, but none of the scarabs.

I've tried putting a scarab in a regular tab, and that shows up correctly and has the correct hover.

@aydjay @thailyn

**Comment by thailyn** [comment]
Ah, it looks like we don't have any `ItemDisplay` in the XAML file for them.  Do you have more information on how the scarabs are laid out?  Right now @aydjay and I don't know what goes where.  Otherwise I could make a PR on top of @aydjay's (since he said he's unavailable for Christmas now) just putting them anywhere, and we can reorganize them as necessary, later.

**Comment by Stickymaddness** [comment]
Ok, from the few scarabs I have, and eyeballing the tab in game with the scarab art on the wiki, I think I've been able to work out all the placements.

As you're probably already aware, there are 12 different types of scarabs (Cartography, Divination, Sulphite, etc) and each of these have 3 quality levels- Rusted, Polished and Gilded.

I'll refer to the fragment tab and try explain the positioning, 

![image](https://user-images.githubusercontent.com/292581/49677643-e834cb80-fa4d-11e8-9338-a11d8af86d4f.png)

The slots for the scarabs have been broken into 3 sections, each 3x4 slots. The columns are repeating in each of these sections, and correspond to the quality levels, from left to right Rusted, Polished and Gilded - the first column of the first section will be rusted, and so will the first column of the second section.

The rows for each section are for the type of scarab, and do _not_ repeat per section - the first row in the first section will be a different type to the first row in the second section.

Now for the placements, below I've listed the type per row, per section. The ones with an asterisk are ones I've guestimated, the rest are ones I can confirm in my tab.


```
Section 1

1 - Bestiary
2 - Reliquary*
3 - Torment*
4 - Sulphite

Section 2

1 Ambush*
2 Shaper
3 Perandus
4 Cartography

Section 3

1 Harbinger*
2 Elder*
3 Divination*
4 Breach

```

Hopefully this is clear enough, and I haven't misplaced any of the ordering!
